gammapy/irf/effective_area.py

```diff
@@ -224,6 +224,25 @@ def find_energy(self, aeff):
                            (self.energy.nodes[[idx - 1, idx]].value))
         return energy * self.energy.unit
 
+    def to_sherpa(self, name):
+        """Return `~sherpa.astro.data.DataARF`
+        
+        Parameters
+        ----------
+        name : str
+            Instance name
+        """
+        from sherpa.astro.data import DataARF
+
+        kwargs = dict(
+            name = name,
+            energ_lo = self.to_table()['ENERG_LO'].quantity.to('keV').value,
```

Comment:
Can you put `table = self.to_table() at the top of this method, to avoid the inefficiency of generating the Table object 3 times?